### ISSUE TITLE
Triggers do not work when install under non-public schema

### ISSUE THREAD

**Issue by nhat-nguyen** [point-source/supabase-tenant-rbac#16]
If I install the extension under a schema different from `public`, creating a new record under `group_users` results in the following error:

```
ERROR:  42P01: relation "group_users" does not exist
QUERY:  UPDATE auth.users
  SET raw_app_meta_data = JSONB_SET(
      raw_app_meta_data,
      '{groups}',
      JSONB_STRIP_NULLS(
        JSONB_SET(
          COALESCE(raw_app_meta_data->'groups', '{}'::JSONB),
          ARRAY[_group_id::TEXT],
          COALESCE(
            (SELECT JSONB_AGG("role")
             FROM group_users gu
             WHERE gu.group_id = _group_id
               AND gu.user_id = _user_id
            ),
            'null'::JSONB
          )
        )
      )
    )
  WHERE id = _user_id
```

The insert query I used is:

```
  insert into supabase_rbac.group_users (group_id, user_id)
  values ('9adaeec1-11c9-411c-966c-ef9b8352cc67', 'a0555121-ffed-4751-9de5-18e5f51cbb69');
```

I think `group_users` needs to be prefixed with the correct schema.

**Comment by point-source** [comment]
Oh good catch. Though I'm not actually sure how to resolve since I'm new to using this TLE / database.dev version of packaging. I will look into other packages to see how they resolve this. There must be a way to detect which schema this gets installed into. Thanks for the report.

**Comment by point-source** [comment]
This is fixed in This should be fixed as of 1.0.0 release
